packages/capabilities/package.json

```
@@ -56,6 +56,10 @@
       "types": "./dist/src/filecoin/dealer.d.ts",
       "import": "./src/filecoin/dealer.js"
     },
+    "./web3.storage/blob": {
+      "types": "./dist/src/web3.storage/blob.d.ts",
+      "import": "./src/web3.storage/blob.js"
+    },
```

We should probably expose other capabilities also

```suggestion
  },
  "./blob": {
      "types": "./dist/src/blob.d.ts",
      "import": "./src/blob.js"
    },
  "./http": {
      "types": "./dist/src/http.d.ts",
      "import": "./src/http.js"
    },
```

---

they are exposes because are in top level https://github.com/web3-storage/w3up/blob/feat/blob-implementation/packages/capabilities/package.json#L32